Tests/com/drew/imaging/jpeg/JpegMetadataReaderTest.java

```diff
@@ -57,6 +59,24 @@ public void testExtractXmpMetadata() throws Exception
         assertNotNull(directory);
     }
 
+    @Test
+    public void testFStopFormatting() throws Exception
+    {
+        Locale previousLocale = Locale.getDefault();
+
+        // Setting this locale would cause an f-number with a comma:
+        Locale.setDefault(Locale.forLanguageTag("nl-NL"));
+
+        try {
+            Metadata metadata = JpegMetadataReader.readMetadata(new File("Tests/Data/withExif.jpg"));
+            final ExifSubIFDDirectory subIFDDirectory = metadata.getFirstDirectoryOfType(ExifSubIFDDirectory.class);
+            assertEquals("f/4.0", subIFDDirectory.getDescription(ExifDirectoryBase.TAG_FNUMBER));
```

Comment:
Thanks for explanation. You can safely close this PR. I'll work around this myself.

Here at work we have a number of different CI build agents, some of which have a different locale than others. This caused some tests to fail.

Overriding the global `Locale` changes it for the entire JVM, which is a bit nasty.

One thing you might consider is allowing to set the `Locale` for `metadata-extractor`. This would then be used for formatting dates and numbers.